pom.xml

```diff
@@ -107,6 +122,15 @@
             <artifactId>environment-reader-library</artifactId>
             <version>${environment-reader-library.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
```

Comment:
Spring-boot-starter-test is defined twice.

Reply:
Nice catch. I'll fix this. Thanks 👍